### PR TITLE
Initial DAOS documentation

### PR DESCRIPTION
**Pull request by kevin-harms**
Adding initial DAOS documentation for temporary oneScratch storage.

### CODE REVIEW

File: docs/aurora/data-management/daos/daos-overview.md [hunk 1]
```diff
@@ -1,3 +1,267 @@
...
+> Total size: 6.0 TB  
+>   Free: 4.4 TB, min:6.5 GB, max:7.0 GB, mean:6.9 GB  
+> - Storage tier 1 (NVMe):  
+>   Total size: 200 TB  
+>   Free: 194 TB, min:244 GB, max:308 GB, mean:303 GB  
+> Rebuild done, 4 objs, 0 recs
+
+## Container
+The container is the basic unit of storage.
+A POSIX container can contain hundres of millions of files, you can use it to store all of your date.
```
Comment (cjknight) [inline docs/aurora/data-management/daos/daos-overview.md:48]: spelling: "hundreds of millions" & "your data"

File: docs/aurora/data-management/daos/daos-overview.md [hunk 2]
```diff
@@ -1,3 +1,267 @@
...
+You only need a small set of containers perhaps just one per major unit of project work.
+
+### Create a container
+ALCF has provided a script, `mkcont`, to help create a container with reasonable defaults.
+You can of course use the `daos` command to create a contaienr directly.
```
Comment (cjknight) [inline docs/aurora/data-management/daos/daos-overview.md:53]: spelling: container

**Review by cjknight — APPROVED** [review]
Looks good.

File: docs/aurora/data-management/daos/daos-overview.md (hunk 1 above)
Comment (kevin-harms) [reply docs/aurora/data-management/daos/daos-overview.md]: corrected

File: docs/aurora/data-management/daos/daos-overview.md (hunk 2 above)
Comment (kevin-harms) [reply docs/aurora/data-management/daos/daos-overview.md]: corrected

**Comment by kevin-harms** [comment]
corrected.

**Pull request merged**